### PR TITLE
Add is_countable polyfill

### PR DESCRIPTION
**Pull request by Samuell1**

### CODE REVIEW

File: src/Support/helpers.php [hunk 1]
```diff
@@ -317,3 +317,15 @@ function http_build_url(array $url, array $replace = [], $flags = HTTP_URL_REPLA
...
 }
+
+if (!function_exists('is_countable')) {
+    /**
+     * Polyfill for `is_countable` method provided in PHP 7.3
+     *
+     * @param  mixed  $var
+     * @return boolean
+     */
+    function is_countable($value) {
```
Comment (bennothommo) [inline src/Support/helpers.php]: @Samuell1 Please put the opening brace on its own line.
```suggestion
    function is_countable($value)
    {
```

**Review by bennothommo — CHANGES_REQUESTED** [review]

**Comment by w20k** [comment]
@Samuell1 maybe you could also add a small Unit test?

**Review by bennothommo — APPROVED** [review]

**Comment by bennothommo** [comment]
@Samuell1 @w20k I've added in some unit tests for this change. I believe that should suffice for testing :)

**Comment by w20k** [comment]
Thanks heaps, @bennothommo! @Samuell1 is a bit lazy today 😃

**Pull request merged**